frontend/src/sdk/usePluginStore.ts

```diff
@@ -0,0 +1,30 @@
+import { PluginLoader, PluginStore } from '@openshift/dynamic-plugin-sdk';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+import useActivePlugins from './useActivePlugins';
+
+// TODO(SDK): Expose
+type ResourceFetch = (url: string, requestInit?: RequestInit, isK8sAPIRequest?: boolean) => Promise<Response>;
```

Comment:
@vojtechszocs not sure how we can expose this out of the SDK... it's in the common repo. Maybe there is some way to infer this from the `PluginLoader`'s options?